### DISCUSSION TITLE
E3V2 Templates vs. Jyers Config - What's the difference?

### DISCUSSION THREAD

**Discussion by charminULTRA** [Jyers/Marlin#1185] [General Discussion]
![image](https://user-images.githubusercontent.com/29433007/126423638-b6576770-1509-4903-b5a5-cccf96e424eb.png)

What's the difference between these two folders? I'm confused because it seems to imply that Jyers Config has something in it that the E3V2 templates do not. Is this just Jyers's personal config folder?

**Comment by Cal-LF** [comment]
Yes it is his setup. If you use it, as some have done, problems with compiling will happen. 
The templates are there for us to use as a starting point when compiling your own FW.
